### DISCUSSION TITLE
Can we have multiple slides in different routes?

### DISCUSSION THREAD

**Discussion by naimulcsx** [slidevjs/slidev#609] [Q&A]
Right now the `slides.md` is the slide that gets rendered in the `/` path. I am wondering if this is possible to host multiple slides within a single repo, e.g. slides-1.md, slides-2.md will get rendered in `/slides-1` and `/slides-2` respectively.

**Comment by fguisso** [comment]
I do this in slidev-cli 0.22 but in the new version I get some errors, but you can look here to get a tip in how to do that: https://github.com/fguisso/talks

**Comment by kxxt** [comment]
I created a template monorepo that could automatically deploy multiple slides to their respective routes and BTW it also generates a nice directory listing.

https://github.com/kxxt/slidev-template-repo

**Comment by luc-hn** [comment]
+1
